include/alpaka/core/Common.hpp

```diff
@@ -90,8 +90,7 @@
 //!     alpaka::memcpy(queue, viewGlobalMemUninitialized, bufHost, extent);
 //! }
 //! \endcode
-#if((BOOST_LANG_CUDA && BOOST_COMP_CLANG_CUDA) || (BOOST_LANG_CUDA && BOOST_COMP_NVCC && BOOST_ARCH_PTX)              \
-    || BOOST_LANG_HIP)
+#if((BOOST_COMP_CLANG_CUDA) || (BOOST_LANG_CUDA && BOOST_COMP_NVCC && BOOST_ARCH_PTX) || BOOST_LANG_HIP)
```

Comment:
I wonder if this can be simplified to

```suggestion
#if(BOOST_LANG_CUDA ||  BOOST_LANG_HIP)
```

@psychocoderHPC Do you know?

Reply:
It cannot.